.coveragerc

```diff
@@ -5,5 +5,7 @@ omit =
 
 [report]
 omit =
+    src/manage.py
+    */tests.py
```

Comment:
yeah, do a coverage check before checking this in. make sure nothing important is without tests